app/questionnaire/questionnaire_schema.py

```diff
@@ -25,6 +25,9 @@ def __init__(self, questionnaire_json, language_code=DEFAULT_LANGUAGE_CODE):
     def is_hub_enabled(self):
         return self.json.get('hub', {}).get('enabled')
 
+    def get_hub_required_section_ids(self):
```

Comment:
This makes me think this is required for the hub and spoke part, rather then is needed before it gets there